README.md

```diff
@@ -1,77 +1,24 @@
 
-Optimizely Rollouts is free feature flags for development teams. Easily roll out and roll back features in any application without code deploys. Mitigate risk for every feature on your roadmap. Learn more at https://www.optimizely.com/rollouts/, or see the [documentation](https://docs.developers.optimizely.com/rollouts/docs).
+Optimizely Rollouts is [free feature flags](https://www.optimizely.com/free-feature-flagging/) for development teams. You can easily roll out and roll back features in any application without code deploys, mitigating risk for every feature on your roadmap.
 
-### Features
+## Get Started
 
-- Automatic datafile downloading
-- User ID + attributes memoization
-- Render blocking until datafile is ready via a React API
-- Optimizely timeout (only block rendering up to the number of milliseconds you specify)
-- Library of React components and hooks to use with [feature flags](https://docs.developers.optimizely.com/full-stack/v4.0/docs/create-feature-flags)
+Refer to the [React SDK's developer documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/javascript-react-sdk) for detailed instructions on getting started with using the SDK.
 
-### Compatibility
-
-The React SDK is compatible with `React 16.3.0 +`
-
-### Example
-
-```jsx
-import {
-  createInstance,
-  OptimizelyProvider,
-  useDecision,
-} from '@optimizely/react-sdk';
-
-const optimizelyClient = createInstance({
-  sdkKey: 'your-optimizely-sdk-key',
-});
-
-function MyComponent() {
-  const [decision] = useDecision('sort-algorithm');
-  return (
-    <React.Fragment>
-      <SearchComponent algorithm={decision.variables.algorithm} />
-      { decision.variationKey === 'relevant_first' && <RelevantFirstList /> }
-      { decision.variationKey === 'recent_first' && <RecentFirstList /> }
-    </React.Fragment>
-  );
-}
-
-class App extends React.Component {
-  render() {
-    return (
-      <OptimizelyProvider
-        optimizely={optimizelyClient}
-        timeout={500}
-        user={{ id: window.userId, attributes: { plan_type: 'bronze' } }}
-      >
-        <MyComponent />
-      </OptimizelyProvider>
-    );
-  }
-}
-```
-
-# Contents
-
-1. [Installation](#installation)
-2. [Usage](#usage)
-3. [Credits](#credits)
-4. [Additional code](#additional-code)
-5. [Contribute to this repo](#contribute-to-this-repo)
-
-# Installation
+### Install the SDK
 
 ```
 npm install @optimizely/react-sdk
 ```
 
-# Usage
+## Use the React SDK
+
+### "Initialization"
 
 ## `createInstance`
 

```

Comment:
Please update "Instantiate" link to: https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/initialize-sdk-react

Please update "JavaScript: Client-side Datafile Management" link to: https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/javascript-client-side-implementation